docs/src/components/ComponentDoc/ComponentSidebar/ComponentSidebar.tsx

```diff
@@ -44,11 +44,11 @@ class ComponentSidebar extends React.Component<any, any> {
     return (
       <Sticky context={examplesRef} offset={15}>
         <Menu as={Accordion} fluid style={sidebarStyle} text vertical>
-          {_.map(sections, ({ examples, sectionName }) => (
+          {_.map(sections, ({ examples, sectionName }, index) => (
             <ComponentSidebarSection
               activePath={activePath}
               examples={examples}
-              key={sectionName}
+              key={`${sectionName}-${index}`}
```

Comment:
Can we move this to a separate PR? It will be hard to find why this change was done in future.

Reply:
@layershifter  there was a console error that about rendering elements with the same key using array/iterator

![screenshot 2018-11-22 at 17 06 51](https://user-images.githubusercontent.com/5442794/48913578-15dc1b00-ee79-11e8-9d7c-0ace7b9a44cd.png)

I cannot open another PR as I have 2 already.. wouldn't this comment here be enough? I can amend the commit comment as well if necessary

Reply:
@layershifter any resolution or can I resolve this?

Reply:
will keep the change in this PR